code/Gerrit.md

```diff
@@ -0,0 +1,297 @@
+# Gerrit
+
+Many developers at the company use a tool called Gerrit (https://review.dev.storj.io/) to submit code and get reviews. There are some key differences between using Github PRs and Gerrit changesets. This document is intended to highlight some of these differences and debug common issues that may arise. It is also a quick source for getting set up with Gerrit for the first time.
+
+## Table of Contents
```

Comment:
The page rendering should add these automatically.

Reply:
Where is the `main` branch published so I can see an example? I thought that the contributor docs were only in this repo and weren't published to some other site that would handle the table of contents

Reply:
<img width="359" alt="Screenshot 2022-05-14 at 22 00 04" src="https://user-images.githubusercontent.com/192964/168444975-b8557e73-00a6-40ce-ac7a-50a446c02ddc.png">

And if you open in storj/storj wiki then it's on the right side

Reply:
done